q2_feature_table/plugin_setup.py

```diff
@@ -197,7 +198,10 @@
         'where': 'SQLite WHERE clause specifying sample metadata criteria '
                  'that must be met to be included in the filtered feature '
                  'table. If not provided, all samples in `metadata` that are '
-                 'also in the feature table will be retained.'
+                 'also in the feature table will be retained.',
+        'exclude_ids': 'If `True`, function returns the filtered set of '
```

Comment:
Sorry, I peeked through some of the other plugins/actions out there in the ecosystem, maybe something like this:

> If `True`, returns the filtered set of samples which occur in the feature table but not in the metadata (or the filtered metadata, if invoked in conjunction with `where`).

Note, we don't need to say anything about the default value --- that is for the interfaces to handle.